inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/ov_infer_request/infer_request_dynamic.cpp

```diff
@@ -18,6 +18,10 @@ const std::vector<std::map<std::string, std::string>> HeteroConfigs = {
     {{"TARGET_FALLBACK", CommonTestUtils::DEVICE_CPU}}
 };
 
+const std::vector<std::map<std::string, std::string>> AutoConfigs = {
+    {{InferenceEngine::MultiDeviceConfigParams::KEY_MULTI_DEVICE_PRIORITIES, CommonTestUtils::DEVICE_CPU}}
```

Comment:
so we expclitly test the AUTO:CPU is working, but again, how about just "AUTO" - will it select the CPU (just because the GPU will fail loading the dynamic network)?
Shall we have such a test

Reply:
add case for AUTO, AUTO:GPU,CPU and AUTO:GPU respectively